docs/user/openstack/install_upi.md

```diff
@@ -552,48 +550,19 @@ To obtain the token execute:
 openstack token issue -c id -f value
 ```
 
-Add a new entity to the list of http headers:
+The command will return the token to be added to the `ignition.config.append[0].httpHeaders` property in the Bootstrap Ignition Shim (see [below](#create-the-bootstrap-ignition-shim)):
 
 ```json
-{
-  "httpHeaders": [
-    {
-      "name": "X-Auth-Token",
-      "value": "<token_id>"
-    }
-  ]
-}
+"httpHeaders": [
+	{
+		"name": "X-Auth-Token",
+		"value": "<token>"
+	}
+]
 ```
 
-The result shim config should look like:
-
-```json
-{
-  "ignition": {
-    "config": {
-      "append": [
-        {
-          "source": "https://public.glance.example.com:9292/v2/images/b7e2b84e-15cf-440a-a113-3197518da024/file",
-          "verification": {},
-          "httpHeaders": [
-            {
-              "name": "X-Auth-Token",
-              "value": "46b30a6a00d654f4af0fc5bb8b8413f1"
-            }
-          ]
-        }
-      ]
-    },
-    "security": {},
-    "timeouts": {},
-    "version": "2.4.0"
-  },
-  "networkd": {},
-  "passwd": {},
-  "storage": {},
-  "systemd": {}
-}
-```
+Combine the public URL with the `file` value from the previous step to get the link to your bootstrap ignition.
+Example of the link: `https://public.glance.example.com:9292/v2/images/b7e2b84e-15cf-440a-a113-3197518da024/file`.
```

Comment:
perhaps it's good to mention this link would need to be added to the source field on the Ignition Shim